src/profile-logic/import/chrome.js

```diff
@@ -707,7 +713,9 @@ async function extractScreenshots(
     thread.markers.name.push(
       thread.stringTable.indexForString('CompositorScreenshot')
     );
-    thread.markers.time.push(screenshot.ts / 1000);
+    thread.markers.startTime.push(screenshot.ts / 1000);
+    thread.markers.endTime.push(screenshot.ts / 1000);
```

Comment:
Shouldn't we push null for `endTime` if it's an instant marker?

Reply:
Ah yeah, we probably should. I don't think there is a behavior change either way, but this would be more correct.